src/js/syntax-highlight.js

```diff
@@ -1,11 +1,15 @@
 import throwError from './helpers';
 
-import prism from 'prism/prism.js';
-import diff from './languages/prism-diff.js';
+import prism from 'prism/components/prism-core.js';
 // Adds to Prism global object which we remove https://github.com/PrismJS/prism/blob/v1.15.0/prism.js#L6
+import 'prism/components/prism-markup.js';
+import 'prism/components/prism-css.js';
+import 'prism/components/prism-clike.js';
```

Comment:
Actually, do we need this?

Reply:
It is part of the default set we had before, if we remove it, it's a breaking change and would be a major version bump. We can do that in another PR if we want to.

Reply:
https://github.com/Financial-Times/o-syntax-highlight/issues/29